docs/develop/customize.md

```diff
@@ -26,28 +33,26 @@ Go to the browser [*https://localhost:5000/*](https://localhost:5000) or refresh
 !!! warning "That evil cache"
     If you do not see it changing, check in an incognito window; the browser might have cached the logo.
 
-Need further customizations? Have you thought of creating your own extensions? How to add them to your InvenioRDM instance is explained in the next section - [here](../extensions/custom.md).
-
 ## Change colors
 
 You might also be wondering: *How do I change the colors so I can make my instance look like my institution's theme?*
 
 We are going to change the top header section in the frontpage to apply our custom background color. Open the `assets/scss/<your instance shortname>/variables.scss` file and edit it as below:
 
-``` console
+``` scss
 $navbar_background_image: unset;
 $navbar_background_color: #000000; // Note this hex value is an example. Choose yours.
 ```
 
-Then, run the `invenio-cli update` command as above and refresh the page! You should be able to see your top header's color changed! You can find all the available variables that you can change [here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/scss/invenio_app_rdm/variables.scss).
+Then, run the `invenio-cli update` command as above and refresh the page! You should be able to see your top header's color changed! You can find all the available styling variables that you can change [here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/scss/invenio_app_rdm/variables.scss).
 
 ## Change search results
 
 Changing how your results are presented in the search page is also something quite common.
 
 We are going to update the search result template so we can show more text in the result's description. Create a file called `ResultsItemTemplate.jsx` inside `assets/templates/search` folder and then edit it as below:
 
-```console
+```js
```

Comment:
jsx is not supported, so the closest to some highlighting we can get is `js` but it doesn't render quite properly because it's JSX (html tags, curly braces...)